### DISCUSSION TITLE
Make Container Work Offline

### DISCUSSION THREAD

**Discussion by gimse** [developmentseed/titiler#1009] [General]
Hi. The fastapi docs does not work without interne. Can we replace it with [fastapi-offline](https://pypi.org/project/fastapi-offline/) ?

**Comment by vincentsarago** [comment]
Hi @gimse

I would like to avoid depending on a `third-party` dependency, meaning that it would happen at one point that fastapi-offline won't be update do date with the official FastAPI. 

That's said, you can easily write your own TiTiler application using FastAPI Offline
